plugins/modules/database_view_info.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: database_view_info
+
+author:
+  - Toni Moreno (@toni-moreno)
+
+short_description: a module that users can use to query on any previously defined database_view
+
+description:
+  - query on any previously defined database_view
+version_added: 1.6.0
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.query
+options:
+  state:
+  database_view:
+    description:
+      - The name of the database view to query
+    type: str
+    required: true
+  return_fields:
+    description:
+      - A list of fields to return
+    type: list
+    required: false
+notes:
+  - Supports check_mode.
+"""
+
+EXAMPLES = r"""
+  - name: Retrieve all entries from the requested view
+    servicenow.itsm.database_view_info:
+      instance:
+        host: https://instance_id.service-now.com
+        username: user
+        password: pass
+      database_view: "u_my_db_view_1"
+      return_fields:
+      - prefix1_sys_id
+      - prefix1_name
+      - prefix2_name
+      - prefix2_attr_1
+      - prefix2_attr_2
+
+- name: Retrieve all entries from the requested view
+  servicenow.itsm.configuration_item_info:
+    database_view: "u_my_db_view_2"
+    query:
+      - prefix1_sys_id: = <some_sys_id>
+        prefix1_atribute_1: = <any_value>
+      - prefix1_sys_id: = <some_sys_id>
+    return_fields:
+      - prefix1_sys_id
+      - prefix1_name
+      - prefix2_name
+      - prefix2_attr_1
+      - prefix2_attr_2
+  register: result
+"""
+
+
+RETURN = r"""
+record:
+  description:
+    - A list of database records.
+    - Note that the fields of the returned records depend on the database view I(database_view) definition.
+    - field names will be prefixed depending on the prefix defined in the database view definition.
+    - be careful each record will have the sum of fields from each single joined table.
+    - use return_fields to limit the fields to be returned.
+  type: list
+  returned: success
+  type: list
```

Comment:
Duplicate of `type: list` element

Reply:
ok

Reply:
Hello @juremedvesek than you very much for you help. I'm new coding ansible modules and , I think I have not build my development environment properly.

When I exec `make sanity` I have this message 
``` 
toni@desktop01:/servicenow.itsm$ make sanity
Makefile:12: *** Place collection at <WHATEVER>/ansible_collections/servicenow/itsm.  Alto.
```
I've seen what is suposed to to  looking the Makefile.
```Makefile
.PHONY: sanity
sanity:  ## Run sanity tests
	pip install -r sanity.requirements
	black -t py27 --check --diff --color plugins tests/unit
	flake8
	ansible-test sanity --docker
```

if I exec `black -t py27 --check --diff --color plugins tests/unit` this is my output.

```